internal/storage/db/mysql/conf.go

```diff
@@ -18,6 +18,8 @@ type Conf struct {
 	ServerPubKey map[string]string      `yaml:"serverPubKey" conf:",example=\n  mykey: testdata/server_public_key.pem"`
 	// DSN is the data source connection string.
 	DSN string `yaml:"dsn" conf:"required,example=\"user:password@tcp(localhost:3306)/db?interpolateParams=true\""`
+	// Verify enables the pre-check for the required database tables.
+	Verify bool `yaml:"verify" conf:",example=false"`
```

Comment:
I think we should default this to `true` because the main reason for introducing this feature is that some people don't RTFM. 

```suggestion
	SkipSchemaCheck bool `yaml:"skipSchemaCheck" conf:",example=true"`
```